packages/codebyte-editor/dist/stories/CodeByteEditor.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { CodeByteEditor } from '../index';
+export default {
+  title: 'Example/Codebytes',
+  component: CodeByteEditor
+};
+
+var Template = function Template(args) {
+  return /*#__PURE__*/React.createElement(CodeByteEditor, args);
+};
+
+export var Default = Template.bind({});
+Default.args = {
+  text: 'hi'
+};
```

Comment:
Let's use the same story format as  Gamut, mdx is awesome and it'll be easier for us to unify tooling between repos.